### PR TITLE
Rel Notes doc text for 4.7.22 release

### PR DESCRIPTION
**Pull request by jeana-redhat**
For release 3 Aug 2021

Preview: https://deploy-preview-35059--osdocs.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-7-release-notes.html#ocp-4-7-22

### CODE REVIEW

File: release_notes/ocp-4-7-release-notes.adoc [hunk 1]
```diff
@@ -2717,3 +2717,19 @@ With this update, `aws-sdk-go` is bumped to v1.38.35, which supports new Amazon
 ==== Upgrading
 
 To upgrade an existing {product-title} 4.7 cluster to this latest release, see xref:../updating/updating-cluster-cli.adoc#update-service-overview_updating-cluster-cli[Updating a cluster by using the CLI] for instructions.
+
+[id="ocp-4-7-22"]
+=== RHBA-2021:2903 - {product-title} 4.7.22 bug fix update
+
+Issued: 2021-08-03
+
+{product-title} release 4.7.22, is now available. The list of bug fixes that are included in the update is documented in the link:https://access.redhat.com/errata/RHBA-2021:2903[RHBA-2021:2903] advisory. The RPM packages that are included in the update are provided by the link:https://access.redhat.com/errata/RHBA-2021:2904[RHBA-2021:2904] advisory.
```
Comment (bobfuru) [inline release_notes/ocp-4-7-release-notes.adoc:2726]: There's no clause here, so you don't need the comma after the version. 

Also, it's just boilerplate being carried over, but I find this sentence hard to understand: "The list of bug fixes that are included in the update is documented..." _what_ is documented? The list, the bug fixes, the update? Could it just say, "The bug fixes included in the update are documented..."? (I'm a bit crotchety about the use of "that are" for clarity/localization in these cases, so by all means feel free to disregard that omission. 😁 )

**Review by bobfuru — COMMENTED** [review]
A nit and a suggestion/opinion but otherwise LGTM!

File: release_notes/ocp-4-7-release-notes.adoc [hunk 2]
```diff
@@ -2717,3 +2717,27 @@ With this update, `aws-sdk-go` is bumped to v1.38.35, which supports new Amazon
...
+
+Space precluded documenting all of the container images for this release in the advisory. See the following article for notes on the container images in this release:
+
+link:https://access.redhat.com/solutions/6228131[{product-title} 4.7.22 container image list]
+
+[id="ocp-4-7-22-features"]
+==== Features
+
+[id="ocp-4-7-22-aws-sdk-version-update"]
+===== Support for new Amazon Web Services (AWS) region
```
Comment (bergerhoffer) [inline release_notes/ocp-4-7-release-notes.adoc:2736]: for new -> for a new ?

**Review by bergerhoffer — COMMENTED** [review]
Just one small suggestion, and then the two advisory links don't work, but I assume that's expected.

**Pull request merged**